custom_components/tuya_local/devices/caldo_up_t_wifi.yaml

```diff
@@ -0,0 +1,56 @@
+name: CALDO UP T WIFI
```

Comment:
Move this name under products, and put a name here that describes the device's purpose.